desktop/lib.go

```diff
@@ -80,15 +81,17 @@ func start() {
 
 	cdir := configDir(&flags)
 	settings := loadSettings(cdir)
-	proClient = proclient.NewClient(fmt.Sprintf("https://%s", common.ProAPIHost), &webclient.Opts{
+	webclientOpts := &webclient.Opts{
 		HttpClient: &http.Client{
-			Transport: proxied.ParallelForIdempotent(),
+			Transport: proxied.Fronted(30 * time.Second),
```

Comment:
Why the change to use a proxied.Fronted http.RoundTripper here..? We want to continue using `proxied. ParallelForIdempotent`

Reply:
Yep, But there was an issue with `ParallelForIdempotent` this make two requests for each call, and the first call fails every time, even with the proxy on or off, I might need to do more research on this, Will next week.